src/lighteval/logging/info_loggers.py

```diff
@@ -487,6 +490,8 @@ def aggregate(self, task_dict: dict[str, LightevalTask], bootstrap_iters: int =
                 except OverflowError:
                     hlog_warn(f"{task_name}, {metric_name} got an OVERFLOW ERROR when aggregating.")
                     metric_result = float("nan")
+                except KeyError:
```

Comment:
Might be worth it adding an `hlog_warn` here too - not 100% sure it would be needed

Reply:
it should not be needed, this only happens if you have a metric with no aggregation function, meaning you will have it logged out in details but not aggregated in the result file

Reply:
this would also really clog up the terminal

Reply:
I just fear that it might eat up logs for badly configured metrics, but it shouldn't happen anyway, so OK with not having it